test/extended/builds/pipeline_origin_bld.go

```diff
@@ -1,161 +1,42 @@
-			// an override to use that images stream and test image
-			var licensePrefix, pluginName string
-			useSnapshotImage := false
-
-			err := oc.Run("create").Args("-n", oc.Namespace(), "-f", jenkinsTemplatePath).Execute()
-			o.Expect(err).NotTo(o.HaveOccurred())
-
-			jenkinsTemplateName := "jenkins-ephemeral"
-
-			// create persistent volumes if running persistent jenkins
-			if jenkinsTemplatePath == jenkinsPersistentTemplatePath {
-				g.By("PV/PVC dump before setup")
-				exutil.DumpPersistentVolumeInfo(oc)
-
-				jenkinsTemplateName = "jenkins-persistent"
-
-				nfspod, pvs, err = exutil.SetupK8SNFSServerAndVolume(oc, 3)
-				o.Expect(err).NotTo(o.HaveOccurred())
-
-			}
 
 			// our pipeline jobs, between jenkins and oc invocations, need more mem than the default
-			newAppArgs := []string{"--template", fmt.Sprintf("%s/%s", oc.Namespace(), jenkinsTemplateName), "-p", "MEMORY_LIMIT=2Gi", "-p", "DISABLE_ADMINISTRATIVE_MONITORS=true"}
-			newAppArgs = jenkins.OverridePodTemplateImages(newAppArgs)
-			clientPluginNewAppArgs, useClientPluginSnapshotImage := jenkins.SetupSnapshotImage(jenkins.UseLocalClientPluginSnapshotEnvVarName, localClientPluginSnapshotImage, localClientPluginSnapshotImageStream, newAppArgs, oc)
-			syncPluginNewAppArgs, useSyncPluginSnapshotImage := jenkins.SetupSnapshotImage(jenkins.UseLocalSyncPluginSnapshotEnvVarName, localSyncPluginSnapshotImage, localSyncPluginSnapshotImageStream, newAppArgs, oc)
-
-			switch {
-			case useClientPluginSnapshotImage && useSyncPluginSnapshotImage:
-				fmt.Fprintf(g.GinkgoWriter,
-					"\nBOTH %s and %s for PR TESTING ARE SET.  WILL NOT CHOOSE BETWEEN THE TWO SO TESTING CURRENT PLUGIN VERSIONS IN LATEST OPENSHIFT JENKINS IMAGE ON DOCKER HUB.\n",
-					jenkins.UseLocalClientPluginSnapshotEnvVarName, jenkins.UseLocalSyncPluginSnapshotEnvVarName)
-			case useClientPluginSnapshotImage:
-				fmt.Fprintf(g.GinkgoWriter, "\nTHE UPCOMING TESTS WILL LEVERAGE AN IMAGE THAT EXTENDS THE LATEST OPENSHIFT JENKINS IMAGE AND OVERRIDES THE OPENSHIFT CLIENT PLUGIN WITH A NEW VERSION BUILT FROM PROPOSED CHANGES TO THAT PLUGIN.\n")
-				licensePrefix = clientLicenseText
-				pluginName = clientPluginName
-				useSnapshotImage = true
-				newAppArgs = clientPluginNewAppArgs
-			case useSyncPluginSnapshotImage:
-				fmt.Fprintf(g.GinkgoWriter, "\nTHE UPCOMING TESTS WILL LEVERAGE AN IMAGE THAT EXTENDS THE LATEST OPENSHIFT JENKINS IMAGE AND OVERRIDES THE OPENSHIFT SYNC PLUGIN WITH A NEW VERSION BUILT FROM PROPOSED CHANGES TO THAT PLUGIN.\n")
-				licensePrefix = syncLicenseText
-				pluginName = syncPluginName
-				useSnapshotImage = true
-				newAppArgs = syncPluginNewAppArgs
-			default:
-				fmt.Fprintf(g.GinkgoWriter, "\nNO PR TEST ENV VARS SET SO TESTING CURRENT PLUGIN VERSIONS IN LATEST OPENSHIFT JENKINS IMAGE ON DOCKER HUB.\n")
-			}
+			newAppArgs := []string{"jenkins-ephemeral", "-p", "MEMORY_LIMIT=2Gi", "-p", "DISABLE_ADMINISTRATIVE_MONITORS=true"}
```

Comment:
all the new-app related hoopla that has been deleted was related to testing on 3.x

not needed for 4.x